{"loop_id": 0, "unhandled_api_list": ["java.util.concurrent.Future.get()"], "num_of_unhandled_api": 1, "unhandled_exception_list": ["Please check if the computation was cancelled for java.util.concurrent.Future.get() in the code snippet you provided, otherwise throw CancellationException."], "num_of_unhandled_exception": 1}
{"loop_id": 1, "unhandled_api_list": ["java.util.concurrent.Future.get()", "java.util.concurrent.Future.get()"], "num_of_unhandled_api": 2, "unhandled_exception_list": ["Please check if the computation was cancelled for java.util.concurrent.Future.get() in the code snippet you provided, otherwise throw CancellationException.", "Please check if the computation threw an exception for java.util.concurrent.Future.get() in the code snippet you provided, otherwise throw ExecutionException."], "num_of_unhandled_exception": 2}
{"loop_id": 2, "unhandled_api_list": [], "num_of_unhandled_api": 0, "unhandled_exception_list": [], "num_of_unhandled_exception": 0}
